test/user.test.js

```diff
@@ -257,8 +257,9 @@ describe('User', function() {
         User.create({ email: 'b@c.com', password: pass72 }, function(err) {
           if (err) return done (err);
           User.login({ email: 'b@c.com', password: badPass }, function(err) {
-            assert(err && !/verified/.test(err.message),
```

Comment:
FWIW, `err.message` was evaluated before lazy evaluation of `&&` kicked in. When `err` was null, the test failed with a wrong error.